nilearn/decoding/searchlight.py

```diff
@@ -23,7 +23,7 @@
 
 from .. import masking
 from ..image.resampling import coord_transform
-from ..input_data.nifti_spheres_masker import _apply_mask_and_get_affinity
+from nilearn.maskers.nifti_spheres_masker import _apply_mask_and_get_affinity
```

Comment:
Why switch to absolute import here?

Reply:
I think this is one of our conventions which is not clearly written but that we try to enforce. See #1919 for example, and also the first comment of @emdupre in #3016.
I agree they look a little weird in the middle of all these relative imports though.

Reply:
Ah, so this is basically a matter of slowly replacing the relative imports with absolute ones, but you're only changing the imports directly affected by the new module. Thanks for clarifying!